apiserver/facades/client/bundle/state.go

```diff
@@ -3,4 +3,15 @@
 
 package bundle
 
-type Backend interface{}
+import (
+	"github.com/juju/juju/migration"
+	"github.com/juju/juju/state"
+)
+
+type Backend interface {
+	migration.StateExporter
```

Comment:
I am not sure that we actually want this interface here (nor this import)... Nor do I think you should change migration interface for the purpose of the bundle export!
 I think it should be just a method declaration  you are after...

```
ExportPartial(cfg state.ExportConfig) (description.Model, error)
```